qemu/tests/virtio_fs_share_data.py

```diff
@@ -75,6 +75,22 @@ def get_stdev(file):
         logging.info("%s device id is %s.", file, stdev)
         return stdev
 
+    def check_socket_group():
+        """
+        check socket path's user group
+        """
+        cmd_get_sock = params["cmd_get_sock"]
+        for device in vm.devices:
+            if isinstance(device, qdevices.QVirtioFSDev):
+                sock_path = device.get_param("sock_path")
```

Comment:
```suggestion
                sock_path = device.get_param("sock_path")
                break
```
We can reduce the loop times.